securedrop_client/logic.py

```diff
@@ -310,6 +311,19 @@ def on_authenticate_failure(self, result: Exception) -> None:
                   'Please verify your credentials and try again.')
         self.gui.show_login_error(error=error)
 
+    def on_get_current_user_success(self, result) -> None:
+        user = storage.update_and_get_user(
+            result['uuid'],
+            result['username'],
+            result['first_name'],
```

Comment:
(expects a 0.14.0 version server or later for running this against hardware instance, which will soon be out, so this is fine, just flagging for awareness)